src/main/java/Task.java

```diff
@@ -0,0 +1,57 @@
+import java.util.*;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Task {
+    private static List taskList = new LinkedList();
+    public static void addList(String task){
+        taskList.add(task);
+    }
+    public static void deleteList(String task){
+        taskList.remove(task);
+    }
+    public static void printList() {
+        Stream stream = taskList.stream();
+        stream.forEach(System.out::println);
+    }
+    public static void printSalute(){
+        System.out.println(
+                        "    Please select one of the following (key in number): \n" +
+                        "    1. Add a new task, \n" +
+                        "    2. Delete an old task, \n" +
+                        "    3. Show my tasks, or\n"+
+                        "    4.bye \n" +
+                        "    to end this conversation");
+    }
+    public static void exe(int instruction){
+        Scanner scan = new Scanner(System.in);
+        switch (instruction) {
+        case 1: //add task
```

Comment:
The magic numbers (1, 2, 3) in the cases could be refactored into constants that check the type of command requested by the user